iac-extensions/terraform/src/test/resources/checks/GCP/AdministrativeRolesOnWorkloadResourcesCheck/role_assignment.tf

```
@@ -0,0 +1,62 @@
+resource "google_apigee_environment_iam_binding" "binding" {
+  org_id = google_apigee_environment.apigee_environment.org_id
+  env_id = google_apigee_environment.apigee_environment.name
+  role  = "roles/ml.jobOwner" # NonCompliant
+  members = [
+    "user:jane@example.com",
+  ]
+}
+
+resource "google_apigee_environment_iam_member" "member" {
+  org_id = google_apigee_environment.apigee_environment.org_id
+  env_id = google_apigee_environment.apigee_environment.name
+  role = "roles/apigee.admin" # NonCompliant
+  member = "user:jane@example.com"
```

```suggestion
  role = "roles/apigee.admin" # Noncompliant
```
1. I guess you have to write `Noncompliant` instead of `NonCompliant` but no bet
2. You can get rid of the overhead because it is not necessary for the check. Having it on the first test case should be enough.

---

`# NonCompliant` works too (tested) but okay - complying